react/components/FormattedPrice.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { FunctionComponent } from 'react'
+import { FormattedNumber } from 'react-intl'
+
+interface Props {
+  value: number
+  currency: string
+}
+
+const FormattedPrice: FunctionComponent<Props> = ({ value, currency }) => (
```

Comment:
maybe use @klzns component instead of this.
https://github.com/vtex-apps/format-currency